app/services/proofing/lexis_nexis/ddp/verification_request.rb

```diff
@@ -0,0 +1,46 @@
+module Proofing
+  module LexisNexis
+    module Ddp
+      class VerificationRequest < Request
+        private
+
+        def build_request_body
+          {
+            api_key: config.api_key,
+            org_id: config.org_id,
+            account_address_street1: applicant[:address1],
+            account_address_street2: applicant[:address2] || '',
+            account_address_city: applicant[:city],
+            account_address_state: applicant[:state],
+            account_address_country: 'US',
+            account_address_zip: applicant[:zipcode].match(/^\d{5}/).to_s,
+            account_date_of_birth: applicant[:dob] ?
+              Date.parse(applicant[:dob]).strftime('%Y%m%d') : '',
+            account_email: applicant[:email],
+            account_first_name: applicant[:first_name],
+            account_last_name: applicant[:last_name],
+            account_telephone: '', # applicant[:phone], decision was made not to send phone
+            drivers_license_number_hash: applicant[:state_id_number] ?
+              OpenSSL::Digest::SHA256.hexdigest(applicant[:state_id_number].gsub(/\W/, '')) : '',
```

Comment:
I'd consider extracting methods for these `_hash` attributes so we could unit test them, which would allow us to check and verify the regexes like `state_id_number_for_hash` or `ssn_for_hash`

Reply:
Agree.  This particular code is covered but it's quick and dirty.